propertyestimator/workflow/decorators.py

```diff
@@ -7,6 +7,8 @@
 
 from enum import Enum
 
+from propertyestimator.workflow.utils import PlaceholderInput
+
 
 class MergeBehaviour(Enum):
```

Comment:
I don't have a particular preference, but we probably need to settle on a single spelling convention. I think the US most common spelling is "behavior" in this case.

Reply:
Definitely agree - good spot!